### PR TITLE
feat: add trace log in filters: identityFilter, authFilter, accessLog…

### LINKED ISSUE #159: Add trace log in filters
### Describe This Problem

Now the tenant information related to the source and the apikey is invisible to the filters. It is hard to figure out detail tenant information when resolving problems. And there is no output source information when an exception is reported.

### Proposal

Add trace/debug logs containing tenant info to the filters（ identityFilter, authFilter, accessLogFilter）

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by jsy1001de**
# Which issue does this PR close?

Closes #159 

# What changes are included in this PR?
Define the trace class in RequestContext，and print the tenant and accessKey from the source of the request.

1. In step2IdentityFilter, print trace log where identity error;
2. In step3AuthFilter, print trace log where auth error;
3. In step4AccessLogFilter, print trace log and show source traffic;

# Are there any user-facing changes?

# How does this change test

### CODE REVIEW

File: server/home/home-common/src/main/java/io/holoinsight/server/home/common/util/retrofit/RetrofitProxy.java [hunk 1]
```diff
@@ -68,15 +68,15 @@ protected void printBefore(Method method, Object[] args) {
...
+    log.warn(invokeInfo.toString());
   }
 
   protected void printAfter(long start, String url, Method method, Response response) {
 
     String invokeInfo = "Invoke INFO: method=[" + method.getName() + "], url=[" + url + "], cost=["
         + (System.currentTimeMillis() - start) + "], " + "result=[" + response.isSuccessful()
         + "], code=[" + response.code() + "]";
-    // logger.warn(invokeInfo);
+    log.warn(invokeInfo);
```
Comment (xzchaoo) [inline server/home/home-common/src/main/java/io/holoinsight/server/home/common/util/retrofit/RetrofitProxy.java:79]: use log placeholder, for example: log.warn(" ... {} ... {}")

Comment (jsy1001de) [reply server/home/home-common/src/main/java/io/holoinsight/server/home/common/util/retrofit/RetrofitProxy.java]: > 
have completed

File: server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step3AuthFilter.java [hunk 2]
```diff
@@ -59,9 +59,9 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         filterChain.doFilter(servletRequest, servletResponse);
       }
     } catch (Throwable e) {
-      resp.sendError(HttpServletResponse.SC_UNAUTHORIZED, "auth check error");
-      log.info("auth info: " + J.toJson(J.toJson(RequestContext.getContext())));
-      log.error("auth check error", e);
+      resp.sendError(HttpServletResponse.SC_UNAUTHORIZED, "auth check error, " + e.getMessage());
+      log.error(RequestContext.getTrace() + " auth check error, auth info: "
```
Comment (xzchaoo) [inline server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step3AuthFilter.java:63]: use log placeholder

File: server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step2IdentityFilter.java [hunk 3]
```diff
@@ -55,8 +56,9 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         filterChain.doFilter(servletRequest, servletResponse);
       }
     } catch (Throwable e) {
-      resp.sendError(HttpServletResponse.SC_UNAUTHORIZED, "identity error");
-      log.error("identity error", e);
+      resp.sendError(HttpServletResponse.SC_UNAUTHORIZED,
+          RequestContext.getTrace() + " identity error, " + e.getMessage());
```
Comment (xzchaoo) [inline server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step2IdentityFilter.java:60]: use log placeholder

File: server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step3AuthFilter.java [hunk 4]
```diff
@@ -79,7 +79,7 @@ public boolean auth(HttpServletRequest req, HttpServletResponse resp) throws Thr
       String token = req.getHeader("apiToken");
       // 接口权限判定
       if (!ulaFacade.authFunc(req) && StringUtil.isBlank(token)) {
-        log.error("no function authed");
+        log.warn("authFunc check failed");
```
Comment (xzchaoo) [inline server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step3AuthFilter.java:82]: Add more detail info about this warn.
Just "authFunc check failed" is not enough to figure out which user/tenant is going wrong.

**Review by xzchaoo — APPROVED** [review]
LGTM

**Pull request merged**